cypress/component/integration/Clipboard.cy.tsx

```diff
@@ -0,0 +1,174 @@
+import '../../../src/global.css';
+import React from 'react';
+import { Clipboard } from '../../../src/components';
+import { compareSnapshot } from '../../compareSnapshot';
+
+describe('Component | Integration | Clipboard', function () {
+  describe('Clipboard', () => {
+    describe('clipboard with only raw-content', () => {
+      beforeEach(() => {
+        cy.mount(<Clipboard rawContent={'raw content'} />);
+      });
+      it('should render', () => {
+        cy.get('[data-testid="clipboard-field"]').should('exist');
+        cy.get('[data-testid="clipboard-content"]').contains('raw content');
+      });
+
+      it('should match snapshot', () => {
+        compareSnapshot(cy, 'clipboard-with-only-raw-content');
+      });
+    });
+
+    describe('clipboard with diplayed content', () => {
+      beforeEach(() => {
+        cy.mount(
+          <Clipboard
+            rawContent={'raw content'}
+            displayedContent={'formatted content'}
+          />,
+        );
+      });
+      it('should render', () => {
+        cy.get('[data-testid="clipboard-field"]').should('exist');
+        cy.get('[data-testid="clipboard-content"]').contains(
+          'formatted content',
+        );
+      });
+
+      it('should match snapshot', () => {
+        compareSnapshot(cy, 'clipboard-with-displayed-content');
+      });
+    });
+
+    describe('clipboard success flow', () => {
+      beforeEach(() => {
+        cy.mount(
+          <Clipboard
+            rawContent={'raw content'}
+            success="Copied!"
+            displayedContent={'formatted content'}
+          />,
+        );
+      });
+      it('should render', () => {
+        cy.get('[data-testid="clipboard-field"]').should('exist');
+        cy.get('[data-testid="clipboard-content"]').contains(
+          'formatted content',
+        );
+      });
+
+      it('should fire the copy clipboard function', () => {
+        cy.get('[data-testid="clipboard-field"]').should('exist');
```

Comment:
You don't need to assert agains the field when you know you will click on it. If it does not exist the click will fail. BTW, you already tested in the previous test its presence.